ow_lander/launch/moveit_rviz.launch

```diff
@@ -1,17 +1,15 @@
 <launch>
 
   <arg name="debug" default="false" />
-  <arg name="use_sim_time" value="true"/>
```

Comment:
Just curious, why was this removed?

Reply:
The whole file should be deprecated actually!

Reply:
This file is replaced by moveit_rviz.launch which is the default name used by moveit setup assistant

Reply:
Ah, I see that now.  Wasn't familiar with the file replacement notation in these PRs.

Reply:
By default **use_sim_time** is not included in that file. For its value to have any effect, the argument needs to be propagated to empty_world.launch. So I need to trace if defining this argument in this file has any effect at all.

Reply:
So on melodic-devel this argument has no purpose in this file, which means removing it is the proper thing

Reply:
Good catch. We definitely want use_sim_time = true to be set when we launch gazebo so that gazebo uses the date stamp we set in the world file.

Reply:
And Devin relied on this parameter for sim speedup, though I'm not sure how it got propagated from ow_autonomy launch files.